### DISCUSSION TITLE
BelongsTo Resource record can be edited when clicking the Edit button in resource details page

### DISCUSSION THREAD

**Discussion by karinaespinola** [laravel/nova-issues#4915] [Q&A] [answered]
- Laravel Version: 9.35.1
- Nova Version: 4.15.2
- PHP Version: 8.1

### Description:
I've got a resource called Question and this one has a relation HasMany with the resource Answer. When I click the  edit button of the related Answer/Options records that is in the table of the Question details page:
![Screen Shot 2022-10-13 at 14 50 13](https://user-images.githubusercontent.com/56842281/195707909-2e5d61d1-f817-4a30-8d56-0f3de73128a6.png)

Then everything is fine because the user can't change the Question to which the Answer/Option belongs to, the question dropdown is readonly:
![Screen Shot 2022-10-13 at 14 54 35](https://user-images.githubusercontent.com/56842281/195708665-9e3dba26-bde8-4ca3-9fb5-cbebf4251993.png)

However when I open the details of an Answer and click the edit button from that page:
![Screen Shot 2022-10-13 at 14 55 47](https://user-images.githubusercontent.com/56842281/195708850-00026912-fff0-417e-a19e-382ffc3336c7.png)

Then the question relation field is not readonly and the user can change the relation. I've solved this using a readonly method on the field when updating. However I thought about bringing it up :) 

I hope it helps! Thank you!

**Comment by crynobone** [comment, marked as answer]
This is an expected behavior when accessing edit page from related detail vs resource detail.
